components/channel_info_modal/channel_info_modal.test.tsx

```diff
@@ -6,15 +6,56 @@ import {Modal} from 'react-bootstrap';
 import {shallow} from 'enzyme';
 
 import {mountWithIntl} from 'tests/helpers/intl-test-helper';
-import ChannelInfoModal from 'components/channel_info_modal/channel_info_modal.jsx';
+import ChannelInfoModal from 'components/channel_info_modal/channel_info_modal';
+import { Channel } from 'mattermost-redux/src/types/channels';
+import { Team } from 'mattermost-redux/types/teams';
 
 describe('components/ChannelInfoModal', () => {
+    let mockChannel: Channel;
```

Comment:
You should be able to use the TestHelper (`utils/test_helper`) to save having to write these fields out. Also, I think it's safe enough to just define the mocks once instead of redefining them for each test.

Reply:
Thanks for pointing that out! I had not idea TestHelper existed. I had originally put the definitions in a beforeEach because I did not know for sure if any of the tests made manipulations to the mock channel / mock team but agree that the beforeEach can be removed.